nix/overlay.nix

```
@@ -55,6 +55,11 @@ in {
     configureFlags = old.configureFlags ++ [ "--enable-prof" "--enable-stats" ];
   });
   broker = final.callPackage ./broker {inherit stdenv; python = final.python3;};
+  simdjson = prev.simdjson.overrideAttrs (old: {
+    cmakeFlags = old.cmakeFlags ++ [
```

From the top of my head:
```suggestion
    cmakeFlags = old.cmakeFlags ++ lib.optionals isStatic [
```